onnx/helper.py

```diff
@@ -232,7 +232,7 @@ def get_attribute_value(attr):
         raise ValueError("Unsupported ONNX attribute: {}".format(attr))
 
 
-def make_tensor_value_info(name, elem_type, shape, doc_string=""):
+def make_tensor_value_info(name, elem_type, shape, doc_string="", shape_denotation=None):
```

Comment:
change `shape_denotation` to `denotation` to match the protobuf field?

Reply:
Ah, I see shape_denotation as `shape.denotation`. Ideally it's `shape.dimension.denotation` but no one likes var name that long...
 
I think it's worth making the namespace clear since the denotation is a sub-attribute here (not describing a tensor directly).

Reply:
ok makes sense